src/UkModulusCheck/Helpers.fs

```diff
@@ -31,12 +31,16 @@ module internal Helpers =
     let (|Not|_|) c input =
         if input <> c then Some c else None
 
+    let loadLines parseLine lines =
+        lines
+        |> Seq.map (fun line -> if String.IsNullOrWhiteSpace line then None else parseLine line)
+        |> Seq.choose id
```

Comment:
You can do map and choose in one pass, using just choose

Reply:
Good spot, fixing it now 👍